kuma/static/js/interactive.js

```diff
@@ -26,9 +44,8 @@
             mdn.analytics.trackEvent({
                 category: event.data.category,
                 action: event.data.action,
-                label: event.data.label,
-                value: event.data.value - performance.timing.fetchStart,
-                hitType: 'timing'
+                label: new Date().getTime() + '-' + randomString(),
```

Comment:
This worked for me in the browser console, generating labels like:

```
"1529419596413-Ij3Cy"
"1529419599067-p4UW5"
"1529419601135-NQtFu"
"1529419601945-tO8nQ"
"1529419602611-PuYzE"
"1529419605323-aUpuQ"
```

Reply:
Perfect! That works for me 👍 :) Thanks guys!

Reply:
👍 Looks great!